### PR TITLE
Check for existence of shortcode ui menu item

### PR DESCRIPTION
**Pull request by joeyblake**
Fixes issue #811 where JS error is thrown attempting to add a gallery after clicking “Add Post Element”

### CODE REVIEW

**Review by sc0ttkclark — APPROVED** [review]

**Review by goldenapples — APPROVED** [review]

**Comment by goldenapples** [comment]
I haven't run across this bug but the code looks good and I'm happy to merge it if other people are hitting it.

**Comment by AriannaStory** [comment]
I can confirm that our environment is hitting this issue, and we'd love a merge. :) @goldenapples

**Comment by goldenapples** [comment]
I'll merge and tag a new release. Thanks for the contribution! 

I haven't actively used this plugin in quite a while, but since apparently it's still being used there really should be a new maintainer. Does anyone actively using it want to step up and become a maintainer?

**Pull request merged**
